code/game/area/areas/ruins/rockplanet.dm

```diff
@@ -6,3 +6,61 @@
 /area/mine/rockplanet_nanotrasen
 	name = "Abandoned Mining Facility"
 
+//Abandoned Mechbay Ruin
```

Comment:
This is way too many unique areas. Lower it down to like 1 or 2.

Reply:
I can cut one or two out. It's fine otherwise.